manifests/init.pp

```diff
@@ -261,6 +261,7 @@
   # host valves
   $singlesignon_valve         = false,
   $accesslog_valve            = true,
+  $accesslog_valve_pattern    = '%h %l %u %t &quot;%r&quot; %s %b',
```

Comment:
This new parameter should be documented:

- in [manifests/config.pp](https://github.com/antoineco/aco-tomcat/blob/a5bad125302585b64e916c8dfadb41e87ce6a7d1/manifests/config.pp#L291)
- in [README.md](https://github.com/antoineco/aco-tomcat/blob/46f44ac9a5f52c9f89a58db2c1628ea195a1b03a/README.md#accesslog_valve)

It should also be added to [manifests/instance.pp](https://github.com/antoineco/aco-tomcat/blob/a5bad125302585b64e916c8dfadb41e87ce6a7d1/manifests/instance.pp), which pretty much duplicates the rest of the class.